prototype_source/inductor_with_x86_cpu_as_backend_of_pt2e.rst

```diff
@@ -0,0 +1,186 @@
+
+    # Create the data, using the dummy data here as an example
+    traced_bs = 50
+    x = torch.randn(traced_bs, 3, 224, 224).contiguous(memory_format=torch.channels_last)
+    example_inputs = (x,)
+
+    # Capture the FX Graph to be quantized
+    with torch.no_grad():
+        # For PyTorch Master, using the API of `capture_pre_autograd_graph`
+        # Note 1: `capture_pre_autograd_graph` is also a short-term API, it will be updated to use the official `torch.export` API when that is ready.
+        exported_model = capture_pre_autograd_graph(
+            model,
+            example_inputs
+        )
+        # Note 2: For PyTorch 2.1 release, please use the API of `torch._dynamo.export` to capture the FX Graph
+        # exported_model, guards = torch._dynamo.export(
+        #     model,
+        #     *copy.deepcopy(example_inputs),
+        #     aten_graph=True,
+        # )
+
+
+Now, we will have the FX Module to be quantized.
+
+2. Apply Quantization Flow
+----------------------------
+
+After we capture the FX Module to be quantized, we will import the Backend Quantizer for X86 CPU and configure how to
+quantize the model.
+
+::
+
+    # Create X86InductorQuantizer
+    quantizer = X86InductorQuantizer()
+    quantizer.set_global(xiq.get_default_x86_inductor_quantization_config())
+
+.. note::
+
+- The default quantization configuration in ``X86InductorQuantizer`` uses 8-bits for both weights and activations, as indicated by the argument ``reduce_range=False``.
+  When Vector Neural Network Instruction is not available, the onednn backend silently chooses kernels that assume
+  `multiplications are 7-bit x 8-bit <https://oneapi-src.github.io/oneDNN/dev_guide_int8_computations.html#inputs-of-mixed-type-u8-and-s8>`_. In other words, potential
+  numeric saturation and accuracy issue may happen when running on CPU without Vector Neural Network Instruction.
+
+After we import the backend-specific Quantizer, we will prepare the model for post-training static quantization.
+``prepare_pt2e`` folds BatchNorm operators into preceding Conv2d operators, and inserts observers in appropriate places in the model.
+
+::
+
+    # PT2E Quantization flow
+    prepared_model = prepare_pt2e(exported_model, quantizer)
+
+Now, we will calibrate the ``prepared_model`` after the observers are inserted in the model.
+
+::
+
+    # We use the dummy data as an example here
+    prepared_model(*example_inputs)
+
+    # Alternatively: user can define the dataset to calibrate
+    # def calibrate(model, data_loader):
+    #     model.eval()
+    #     with torch.no_grad():
+    #         for image, target in data_loader:
+    #             model(image)
+    # calibrate(prepared_model, data_loader_test)  # run calibration on sample data
+
+Finally, we will convert the calibrated Model to a quantized Model. ``convert_pt2e`` takes a calibrated model and produces a reference quantized model.
+
+::
+
+    converted_model = convert_pt2e(prepared_model).eval()
```

Comment:
why are we calling eval here?

after capture, we will disable train/eval calls since it doesn't do the same thing as eager mode API

Reply:
eval should be called before export actually

Reply:
Thanks, I think we can remove it now.